src/modules/job-manager/restart.c

```diff
@@ -197,6 +197,8 @@ static int restart_map_cb (struct job *job, void *arg, flux_error_t *error)
                    (uintmax_t)job->id);
         return -1;
     }
+    if (ctx->max_jobid < job->id)
```

Comment:
Given the previous rename of "checkpoint" to "restart", should/could the commit message here be updated too?